wordpress-mobile/WordPress-iOS
# Reader: Adds new fields to ReaderPost. Bumps model version.

Refs #16599 
This PR adds two new fields to the ReaderPost model. `canSubscribeComments` and `isSubscribedComments`. These values reflect whether the post in question supports subscribing to comments via email, and the current user's subscription status.  This change allows the subscription status to be stored directly on the post rather than fetched separately and will be supported by pending changes to the API. Subsequent PRs will introduce the functionality described in #16599. 

To test:
- Confirm core data is set to the correct model revision.
- Be logged into the app prior to testing this branch. 
- Run this branch and confirm you are still logged in. 
- Smoke test the reader post list and confirm nothing goes boom.

@ScoutHarris could I trouble you for a review? 

## Regression Notes
1. Potential unintended areas of impact
n/a

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

3. What automated tests I added (or what prevented me from doing so)
n/a
PR submission checklist:
